public_html/templates/footer.html

```diff
@@ -1,39 +1,45 @@
 <footer>
-            <div class="footer" style="margin-top:100px">
-            <p>Copyright  2016 - TNS. All Rights Reserved</p>
+            <div class="footer" style="margin-top:75px">
                 <div class="row">
                     <div class="third">
-                        <h3 class="footer-header">Customer Service</h3>
-                        <div><a href="comingsoon.html">Frequently Asked Questions</a></div>
-                        <div><a href="comingsoon.html">Returns</a></div>
+                        <h3 class="footer-header">Site Map</h3>
+                        <div><a href="comingsoon.html">Women</a></div>
+                        <div><a href="comingsoon.html">Men</a></div>
                         <div><a href="comingsoon.html">Live Chat</a></div>
                         <div><a href="comingsoon.html">Track Orders</a></div>
-                        <div><a href="contact.html">Contact Us</a></div>
                     </div>
                     <div class="third">
                         <h3 class="footer-header">Company Info</h3>
                         <div><a href="about.html">About Us</a></div>
-                        <div><a href="comingsoon.html">Complain about Us</a></div>
+                        <div><a href="contact.html">Contact Us</a></div>
+						<br>
+						<form>
+							Join our mailing list!<br>
+							<input type="text" name="donothing">
+							<input type="submit" value="Subscribe"
+						</form>
+						
                     </div>
                     <div class="third">
                         <h3 class="footer-header">Connect</h3>
                         <div>
                             <a href="https://twitter.com/justinbieber">
-                                <img src="assets/images/twitter.png" width="20" height="auto"/>
+                                <img src="assets/images/twitter.png" width="20" height="auto"/>     Twitter
                             </a>
                         </div>
                         <div>
                             <a href="https://www.facebook.com/zuck">
-                                <img src="assets/images/facebook.png" width="20" height="auto"/>
+                                <img src="assets/images/facebook.png" width="20" height="auto"/> Facebook
                             </a>
                         </div>
                         <div>
                             <a href="https://www.instagram.com/kang_gary8888/">
-                                <img src="assets/images/instagram.png" width="20" height="auto"/>
+                                <img src="assets/images/instagram.png" width="20" height="auto"/> Instagram
                             </a>
                         </div>
                     </div>
                 </div>
+			<p style="padding-top:20px;">Copyright  2016 - TNS. All Rights Reserved</p>
```

Comment:
remove inline styling and give it a class

say `<p class="copy-right">...</p>`

and in style.css
add
`.copy-right { text-align:center };`